code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -260,6 +260,8 @@
 		return power * 0.5
 	else if(M.IsSleeping())
 		return power * 0.25
+	else if(ispreternis(M) || isipc(M)) //ipc and preternis don't get round removed
+		return power * 0
```

Comment:
Ah yes, `power * 0`. Also known as just `0`

Reply:
YOU DON'T GET IT, MY BRAIN IS SMOL AND THINKING HURTS